lib/sorts/quick-sort/quick-sort.test.js

```diff
@@ -1,4 +1,3 @@
-
 const should = require('should');
 const algo = require('../../index.js');
```

Comment:
Can your algorithm sort numbers less than 0?

Reply:
Can you sort strings?

Reply:
Cant you sort in descend order?

Reply:
According to your questions, my algorithm can sort numbers less than 0 and strings. 
All necessary tests were added and passed. 
But you are right it can't sort in the descend order.